vm/src/frame.rs

```diff
@@ -576,16 +576,6 @@ impl Frame {
                 self.push_value(rustfunc);
                 Ok(None)
             }
-            bytecode::Instruction::StoreLocals => {
```

Comment:
I faced issues when implementing proper dicts here, so its good that this instruction is gone.

Reply:
Yeah, I thought you might like that. :)

I still think it won't help as much as you'd like, but I hope this will make a few things easier.